pkg/tenant/api/v1/api.go

```diff
@@ -240,6 +241,21 @@ type Member struct {
 	LastActivity      string   `json:"last_activity,omitempty"`
 }
 
+// Normalize performs some cleanup on the Member fields to ensure that fields provided
+// in the JSON request can be used in comparisons and uniqueness checks.
+func (m *Member) Normalize() {
+	m.Email = strings.TrimSpace(m.Email)
+	m.Name = strings.TrimSpace(m.Name)
+	m.Organization = strings.TrimSpace(m.Organization)
+	m.Workspace = strings.ToLower(strings.TrimSpace(m.Workspace))
+	m.ProfessionSegment = strings.TrimSpace(m.ProfessionSegment)
+	for i, s := range m.DeveloperSegment {
+		m.DeveloperSegment[i] = strings.TrimSpace(s)
```

Comment:
Do we have an enum for the developer segments?

Reply:
We don't have one currently, should I create a story for that and the profession segment?

Reply:
Yes please, I think that's a good idea.